apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/env/config/BlockchainProperties.java

```diff
@@ -23,7 +27,7 @@
     private ShardingSettings shardingSettings;
     private ConsensusSettings consensusSettings;
 
-    @JsonCreator
+//    @JsonCreator
```

Comment:
Yes, that is needed for unit tests *ChainsConfigLoaderTest* , *BlockchainConfigUpdaterTest* where JSON is read from file. Otherwise incomplete BlockchainProperties constructor is called without reading internal ShardingSettings + ConsensusSettings json data.

Reply:
It is a weak reason to do so, you can update your tests to use previous constructor

Reply:
I don't need previous constructor, inside BlockchainConfigUpdaterTest I need constructor with ALL properties. I don't need previous constructor (without shardSettings, consensusSettings), which was used only inside ChainsConfigLoaderTest.